spec/ace.md

```diff
@@ -49,6 +52,22 @@ Volumes that are specified in the Pod Manifest are mounted into each of the apps
 For example, say that the worker-backup and reduce-worker both have a `mountPoint` named "work".
 In this case, the executor will bind mount the host's `/opt/tenant1/work` directory into the `path` of each of the matching "work" `mountPoint`s of the two app filesystems.
 
+If the target `path` does not exist in the rendered filesystem, it SHOULD be created, including any missing parent directories.
+If the target `path` is a non-empty directory, its contents SHOULD be discarded (e.g. obscured by the bind mount).
+If the target `path` refers to a file, the ACE SHOULD remove that file and create a directory in its place.
+If multiple targets have overlapping target `path`s (for example, if one is nested within another), the ACE SHOULD consider this an error.
+
+The target `path` directories that the ACE creates SHOULD be owned by UID 0 and GID 0, and have access mode `0755` (`rwxr-xr-xr-x`).
+The ACE implementation MAY provide a method for administrator to specify different permissions on a per-pod basis.
+
+The ACE SHOULD NOT create any paths in the host file system, and MUST consider missing volume source paths an error.
+If the ACE does modify the host file system, it SHOULD be possible to disable this behaviour.
+The ACE MAY implement single-file volumes if the underlying operating system supports it.
+
+If the host volume's `source` path is a symbolic link, the ACE SHOULD consider it an error, and SHOULD NOT attempt to use this link's target as volume.
+The ACE MAY also consider it an error if any intermediate directory in volume's `source` path is a symbolic link.
```

Comment:
I think it would be preferable to keep 67, 68, and 69 as either all permissive or all strict (should or must respectively). If I have decided that it is ok for my executor to allow symbolic links, why must I also implement a way to turn that off?

If there is some rational here for mixing them that I don't see...I would still suggest changing 68 to the ever-so-slightly more forceful "should"...i.e. all symlinks whether parent or end-node _should_ be considered an error.